README.md

```diff
@@ -4,20 +4,55 @@
 
 The JavaScript API for Office enables you to create web applications that interact with the object models in Office host applications. Your application will reference the office.js library, which is a script loader. The office.js library loads the object models that are applicable to the Office application that is running the add-in.
 
-The NPM package for Office.js is a copy of what gets published to the official "evergreen" Office.js CDN, at **<https://appsforoffice.microsoft.com/lib/1/hosted/office.js>**.  The NPM also offers alpha and beta versions for faster-cadence beta-testing (relative to the slower-cadence [official BETA endpoint](https://appsforoffice.microsoft.com/lib/beta/hosted/office.js)).
+<br />
+
+## About the NPM package
+
+The NPM package for Office.js is a copy of what gets published to the official "evergreen" Office.js CDN, at **<https://appsforoffice.microsoft.com/lib/1/hosted/office.js>**. 
+
+While the Office.js CDN contains all currently available Office.js APIs at any moment in time, each version of the NPM package for Office.js contains only the Office.js APIs that were available at the point in time when that version of the NPM package was created. 
+
+### Target scenarios
+
+The NPM package for Office.js is intended as a way for you to obtain your own (non-CDN) copy of the Office.js files, which you can then statically serve from your own site instead of using the CDN. This NPM package is primarily provided to address the following scenarios:
+
+1. If you are developing an add-in behind a firewall, where accessing the Office.js CDN is not possible.
+
+2. If you need offline access to the Office.js APIs (for example, to facilitate offline debugging).
+
+Additionally, using the alpha and beta versions of the NPM package can facilitate faster-cadence beta-testing (relative to the slower-cadence beta CDN at **<https://appsforoffice.microsoft.com/lib/beta/hosted/office.js>**).
```

Comment:
There is no "alpha".  And actually, the beta may or may not be faster-cadence (recently it's been the opposite).  I would either remove this sentence, or reframe as more of a "Additionally, if you're working with someone at Microsoft who recommends you use a specific private "adhoc" build of Office.js for testing, the NPM packages that get published off of this repo will help you easily test that code".